packages/plugin-npm-cli/sources/commands/npm/publish.ts

```diff
@@ -48,8 +48,26 @@ export default (clipanion: any, pluginConfiguration: PluginConfiguration) => cli
 
     // We store it so that TS knows that it's non-null
     const ident = workspace.manifest.name;
+    const version = workspace.manifest.version;
 
     const report = await StreamReport.start({configuration, stdout}, async report => {
+      // Not an error if --tolerate-republish is set
+      if (tolerateRepublish) {
+        const registryData = await npmHttpUtils.get(npmHttpUtils.getIdentUrl(ident), {
```

Comment:
this implies you can't use `--tolerate-republish` for the initial publish of a package, is that by design?

Reply:
Good point, I need to add a guard against 404s 👍